README.md

```diff
@@ -19,35 +28,31 @@ The list of RFCs currently under discussion can be found at: [Active RFCs][].
 
 ## Reviewing RFCs
 
-Each week the [DevTools RFC team][] will attempt to review some set of open RFC issues. Reviewed RFCs will also be mentioned in the weekly DevTools team meeting (see archives for [devtools-weekly-meetings][]).
+Each week the [DevTools RFC team][] will attempt to review open RFC issues. Reviewed RFCs will also be mentioned in the weekly DevTools team meeting (see archives for [devtools-weekly-meetings][]).
 
 The team will send an email a few days prior to the review meeting to [dev-developer-tools][] to announce which RFCs are likely to be reviewed, and encourage peers to comment if needed.
```

Comment:
Good question, I was also wondering what to do about this paragraph. I was thinking maybe we could replace it with something like 'we will bring to the team's attention the open RFCs during the meeting', what do you think? I think it'll be useful so that relevant peers will be made aware that way. For example if a contributor wants to do a change but does not know who to contact, we as insiders should know.

Reply:
Yes that's more in line with what usually happens 👍

Reply:
I updated the doc with the note that we will bring the RFCs up during the meeting, let me know if it looks all fine. resolving this conversation for now